Base/Python/slicer/util.py

```diff
@@ -3795,6 +3795,71 @@ def pip_uninstall(requirements):
     _executePythonModule('pip', args)
 
 
+def validate_module_version(
+    module_name: str,
+    minimum_version: str,
+    attempt_fix: bool = False,
+) -> None:
+    """
+    Ensure that some minimum package requirement is met at runtime.
+
+    If the minimum version requirement is not met, either throw or
+    attempt to install the package and possibly restart Slicer.
+
+    :param module_name: The Python Package Index module to check.
+    :param minimum_version: The minimum version requirement.
+    :param attempt_fix: Whether to attempt to fix an installation or version issue
+        with reinstallation and possible application restart.
+
+    :raises ModuleNotFoundError: if a module satisfying version requirements
+        is not installed and the issue could not be fixed.
+    """
+    import importlib
+    from packaging import version
+
+    needRestart = False
+    needInstall = False
+
+    try:
+        module = importlib.import_module(module_name)
+    except ModuleNotFoundError as e:
+        install_message = f"The current operation requires {module_name}.\nClick OK to install from PyPI."
```

Comment:
Add translation - see https://github.com/Slicer/SlicerLanguagePacks/blob/main/DevelopersManual.md#preparing-python-source-files-for-translation